src/messaging/ExchangeMgr.cpp

```diff
@@ -236,7 +236,14 @@ void ExchangeManager::OnMessageReceived(const PacketHeader & packetHeader, const
                         packetHeader.GetDestinationGroupId().Value());
     }
 
-    // Do not handle unsolicited messages on a inactive session.
+    // Do not handle messages that don't match an existing exchange on a
+    // inactive session, since we should not be creating new exchanges there.
```

Comment:
```suggestion
    // Do not handle messages that don't match an existing exchange on an
    // inactive session, since we should not be creating new exchanges there.
```

Reply:
Done.